erpnext/templates/includes/sign_document.js

```diff
@@ -0,0 +1,84 @@
+$(document).ready(function () {
+	let body = $('.third-party-sign');
+	let $sigdiv = $("#signature");
+
+	// init the jSignature widget
+	$sigdiv.jSignature({
+		height: 300,
+		width: (body.width() - 5),
+		lineWidth: 3
+	});
+
+	// clear the canvas and re-render the decor on it
+	$sigdiv.jSignature("reset");
+
+	$("#signeeDetails input[type='radio']").click(function () {
+		if ($(this).data("party-type") === "Company") {
+			$(".party-type-company").show();
+		} else {
+			$(".party-type-company").hide();
+		}
+	});
+
+	$(".refresh-signature").on("click", function () {
+		$sigdiv.jSignature("reset");
+	});
+
+	$("#approveDocument").on("click", function () {
+		let sign = $sigdiv.jSignature("getData");
+		let signee = $("#signee").val();
+		let party_business_type = $("#signeeDetails input[name='type']:checked").val();
+		let designation;
+
+		if (party_business_type === "Individual") {
+			designation = "NA";
+		} else {
+			designation = $("#signee-designation").val();
+		}
+
+		// proceed only if user has put name, signature and designation
+		if (!(signee && $sigdiv.jSignature("getData", "native").length > 0 && designation)) {
+			frappe.throw(__("Please enter your name, signature and designation"));
+		}
+
+		$(".user-signature").hide();
+		frappe.call({
+			method: "erpnext.utils.authorize_document",
```

Comment:
The path is not proper, co-ordinate with Vishal for the correct path

Reply:
file path is fixed as per dependant PR